pkg/controller/certificaterequests/acme/acme_test.go

```diff
@@ -202,11 +202,31 @@ func TestSign(t *testing.T) {
 				CertManagerObjects: []runtime.Object{baseCRNotApproved.DeepCopy(), baseIssuer.DeepCopy()},
 			},
 		},
-		"a CertificateRequest with a denied condition should do nothing": {
+		"a CertificateRequest with a denied condition should update Ready condition with RequestDenied": {
 			certificateRequest: baseCRDenied.DeepCopy(),
 			builder: &testpkg.Builder{
 				KubeObjects:        []runtime.Object{},
 				CertManagerObjects: []runtime.Object{baseCRDenied.DeepCopy(), baseIssuer.DeepCopy()},
+				ExpectedEvents: []string{
+					"Warning RequestDenied The CertificateRequest was denied by an approval controller",
```

Comment:
Hm - I am guessing this Event comes about because everything set as part of a ready condition is logged as an Event? I'd prefer/expect to see the *approval* controller logging an event to say that the request has been denied (if it feels that is appropriate), and *not* see an Event here which is ultimately just re-stating the Ready condition status